.drone.yml

```diff
@@ -40,48 +40,21 @@ steps:
           -D clippy::wildcard_imports -D clippy::cast_lossless 
           -D clippy::manual_string_new -D clippy::redundant_closure_for_method_calls 
           -D clippy::unused_self
+          -A clippy::uninlined_format_args
       - cargo clippy --workspace --all-features -- -D clippy::unwrap_used
-
-  - name: cargo test
-    image: clux/muslrust:1.64.0
-    environment:
-      LEMMY_DATABASE_URL: postgres://lemmy:password@database:5432/lemmy
-      LEMMY_CONFIG_LOCATION: ../../config/config.hjson
-      RUST_BACKTRACE: 1
-      RUST_TEST_THREADS: 1
-      CARGO_HOME: .cargo
-    commands:
-      - apt-get update
-      - apt-get -y install --no-install-recommends postgresql-client protobuf-compiler libprotobuf-dev
-      - cargo test --workspace --no-fail-fast --all-features
-
-  - name: check defaults.hjson updated
-    image: clux/muslrust:1.64.0
-    environment:
-      CARGO_HOME: .cargo
-    commands:
+      - echo "check with different features"
+      - cargo check --workspace --no-default-features
+      - cargo check --workspace --all-features
```

Comment:
Remove these cargo check commands and instead put:
```
cargo check --package lemmy_utils
cargo check --package lemmy_db_schema
cargo check --package lemmy_db_views
cargo check --package lemmy_db_views_actor
cargo check --package lemmy_db_views_moderator
cargo check --package lemmy_api_common
cargo check --package lemmy_api
cargo check --package lemmy_api_crud
cargo check --package lemmy_apub
cargo check --package lemmy_routes
cargo check --package lemmy_server
cargo check --package lemmy_server --all-features
cargo check --package lemmy_server --no-default-features
```

That way we ensure that every package compiles with default features, and that lemmy_server compiles with different feature flags.

Reply:
Done